### ISSUE TITLE
Ender IO Enhanced Machines Extremely Slow

### ISSUE THREAD

**Issue by Numorphan** [Discomanco/Enigmatica2ExpertSkyblock#70]
#### Issue Description

The enhanced machine tier from Ender IO doesn't seem to be buffed at all.
Alloy Smelter w/ Basic Capacitor: 500 rf/tick
Enhanced Alloy Smelter w/ Basic Capacitor: 40 rf/tick (with correspondingly slower operation time)

Same with SAG Mill, and likely everything else, but not sure about that.

#### Modpack version (Do not use "latest")

v1.2

**Comment by NillerMedDild** [comment]
Nerf the regular and buff the Enhanced machines imo, that's what I'm going to do atleast. If you prefer, you can merge the config from E2E when I update.

**Comment by Numorphan** [comment]
If you don't mind me asking, what part of the config corresponds to the speed of the machines? When I posted this issue, I tried to go and change it for myself. So, I took a look through the Ender IO config files, and the only place that seemed reasonable for it was the capacitor_machines.xml file, but that had entries for the enhanced machines as well, so I'm not sure where to change machine speed as a hotfix/customization for my server.
